processed 12 tasks

task 1 'run-graphql'. lines 6-15:
Error: error sending request for url (http://127.0.0.1:8000/): error trying to connect: tcp connect error: Connection refused (os error 61)

task 2 'run-graphql'. lines 17-26:
Error: error sending request for url (http://127.0.0.1:8000/): error trying to connect: tcp connect error: Connection refused (os error 61)

task 3 'run-graphql'. lines 28-37:
Error: error sending request for url (http://127.0.0.1:8000/): error trying to connect: tcp connect error: Connection refused (os error 61)

task 4 'run-graphql'. lines 39-48:
Error: error sending request for url (http://127.0.0.1:8000/): error trying to connect: tcp connect error: Connection refused (os error 61)

task 5 'run-graphql'. lines 50-60:
Error: error sending request for url (http://127.0.0.1:8000/): error trying to connect: tcp connect error: Connection refused (os error 61)

task 6 'run-graphql'. lines 62-77:
Error: error sending request for url (http://127.0.0.1:8000/): error trying to connect: tcp connect error: Connection refused (os error 61)

task 7 'run-graphql'. lines 79-102:
Error: error sending request for url (http://127.0.0.1:8000/): error trying to connect: tcp connect error: Connection refused (os error 61)

task 8 'run-graphql'. lines 104-121:
Error: error sending request for url (http://127.0.0.1:8000/): error trying to connect: tcp connect error: Connection refused (os error 61)

task 9 'publish'. lines 123-140:
created: object(9,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 4461200,  storage_rebate: 0, non_refundable_storage_fee: 0

task 10 'create-checkpoint'. lines 142-142:
Checkpoint created: 1

task 11 'run-graphql'. lines 144-152:
Response: {
  "data": null,
  "errors": [
    {
      "message": "Internal error occurred while processing request: Error calculating layout for 0xd7a98a652a0137376efd1a045dff3c52cc8690b2dee636f149e3b8a2a8428d6b::m::S1<u32>: Type layout nesting exceeded limit of 128",
      "locations": [
        {
          "line": 4,
          "column": 9
        }
      ],
      "path": [
        "type",
        "layout"
      ],
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR"
      }
    }
  ]
}
